src/app/SharpRaven/Data/ExceptionFrame.cs

```diff
@@ -219,5 +223,11 @@ public override string ToString()
 
             return sb.ToString();
         }
+
+        private static bool IsSystemModuleName(string moduleName)
+        {
+            return !string.IsNullOrEmpty(moduleName) &&
+                moduleName.StartsWith("System.", System.StringComparison.Ordinal);
```

Comment:
Good point.

Are you OK with me just adding `Microsoft.` to the check for now, and coming back later to make it configurable if it's requested?

I guess to make it configurable we could pass in a ISet<string> of known internal prefixes? Or we could allow the user to pass in a regex or a Func<string, bool> that we use to filter the frames?